### ISSUE TITLE
App Lifecycle: Create separate development and production databases

### ISSUE THREAD

**Issue by kevinp** [DrivingThru/DrivingThruiOS#120]
When we deploy the app into production/qa, we should update our api/connection strings to point to a production database.  Ideally this is automated, and part of the CD pipeline

**Comment by vfxdrummer** [comment]
I am creating /dev and /prod root nodes in the database.
The db we are using now has no root node. Once we have safely transitioned, it will be deleted.

**Comment by vfxdrummer** [comment]
I was able to export the db data to a json file and duplicate it.

**Comment by vfxdrummer** [comment]
Next step will be to break dev & prod into separate Firebase projects.
I would like to keep this as a two step system.
First, separate root nodes in the same project, then separate projects.
The latter only really makes sense once production starts to scale.

**Comment by vfxdrummer** [comment]
Ok, thought about this some more. Read a lot about it last night.
I think that the cleanest and best way to go is to have them in separate projects, they are each easily available from the dashboard. This will keep all the analytics and data cleanly separated. The only tricky part is loading / renaming the google .plist file for the current target runnng. But, I've set it up before and can look at an old project. It's just the best route to go.

Luckily, it's simple to export and import data to and from databases in Firebase. I played with it last night. So copying our data to the new prod project is simple. I'll get that fired up, today.
